src/makeRobot.js

```diff
@@ -38,7 +38,75 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${
+        this.version
+      }, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value = 1) {
+      if (value < 0) {
+        return this;
+      } else {
+        this.coords.y += value;
+
+        return this;
```

Comment:
`return this;` don't repeat yourself, fix in all cases
```suggestion
      if (value > 0) {
        this.coords.y += value;
      }
 
      return this;
```